Level-Editor-Guide.md

```diff
@@ -0,0 +1,258 @@
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/select-mode1.png?raw=true)
+  The **Green-Selection** allows placing multiple instaces of a tile or group of tiles by holding <kbd>Left-Mouse</kbd>.
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/select-mode2.png?raw=true)
+  The **Fill-Bucket** let's you fill an area with the selected tile.
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/rubber.png?raw=true)
+  The **Eraser** simply removes tiles and objects from your level.
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/arrow.png?raw=true)
+  The **Selection** tool is used to select objects from your level.
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/move-mode1.png?raw=true)
+  The **Duplicator** can be used to duplicate an object on the spot.
+- ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/path_node.png?raw=true)
+  The **Pathnode** tool allows drawing pathnodes for moving objects, see [Using Pathnodes](#Using-Pathnodes).
+
+Unlike the other tools you only find the Pathnode tool under `Objects` → `Environment`.
+
+
+Tiles And Tilemaps
+==================
+
+Let's make a basic level. First, select the drop down menu labeled `Tiles` in the top right corner and select
+a category of your choosing.
+
+Select the tile you would like to use by clicking on it. You can select multiple tiles at once by holding the
+Left-mouse button and dragging your mouse over the tiles you want to select. Your selection is now displayed
+besides the cursor, moving across the grid. By left-clicking you can place them now as many times as you want.
+
+**Note: Right-clicking on a tile inside your level, you can copy it. This can save you time if you don't want to
+frequently switch back and forth between categories.**
+
+Tilemaps
+--------
+
+To make sure Tux is able to stand on the tiles you place must make sure that you are placing them on a solid tilemap.
+
+A tilemap is where all tiles of a level are drawn. They help organizing your tiles and allow for using multiple layers.
+You can add a tilemap through the `Sector` category in the Objects menu. The default solid tilemap is marked `0` in
+the bottom bar. You can edit pre-existing tilemaps by right-clicking on their icon.
+
+![](images/editor/interactive_layer.png "Sector category of the 'Objects' menu")
+
+Every newly created level comes with three tilemaps by default. One for background elements, one for forground elements
+and one for solid ground. Their marked number is defined by their Z-postion. Is it set below 50, it is behind Tux and
+everything above 50 has it be in front of Tux. The default background tilemap is marked `-100` and the default foreground
+tilemap is marked `100`.
+
+In the tilemap settings you can change their size, their solidity, Z-position, as well as colour you the tiles inside
```

Comment:
Changed this one a bit. Tell if it is better now.

Reply:
Replace "the size" with "their size" so that it is clear that size, solidity and Z-position refer to the tilemap, not the tiles. Replace "colour" with "color" because the rest of the document is in American English.

Reply:
Fixed that. Anything else?

Reply:
The sentence looks good now.